Languages/Polish/DefInjected/MainButtonDef/MedicalInfo.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+  <MedicalTab.label>Medycyna</MedicalTab.label>
```

Comment:
I'd like to get your attention here @FluffierThanThou.
I've used word "Medycyna" for tab name which stands for "Medicine" in the meaning of "the science or practice of the diagnosis, treatment, and prevention of disease".
I thought that this noun fits this better than "Medyczne", what is translation for Medical. In Polish it sounds strange as a tab name.

I can change it of course. Write me back what you think about it.

Cheers :)

Reply:
I'll go with your judgment on this :)